client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -60,22 +60,16 @@ class SettingsPayments extends Component {
 
 	onSave = () => {
 		const { translate, site, finishedInitialSetup } = this.props;
-
-		let successAction = successNotice(
-			translate( 'Payment settings saved.' ),
-			{ duration: 4000 }
-		);
-
-		if ( ! finishedInitialSetup ) {
-			successAction = () => {
+		const successAction = () => {
+			if ( ! finishedInitialSetup ) {
 				page.redirect( getLink( '/store/:site', site ) );
+			}
 
-				return successNotice(
-					translate( 'Payment settings saved.' ),
-					{ duration: 4000, displayOnNextPage: true }
-				);
-			};
-		}
+			return successNotice(
+				translate( 'Payment settings saved.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Payment Settings' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).